### ISSUE TITLE
The deliveries gateway is down

### ISSUE THREAD

**Issue by Willow-Systems** [oonqt/pebble-deliveries#5]
**Describe the bug**
Any requests to deliveries.memester.xyz return a 502 from cloudflare, presumably the backend is down. This breaks the watchapp

**Expected behavior**
The package details should be returned from the endpoint

**Screenshots**
Pebble logs 
![image](https://user-images.githubusercontent.com/24474651/120602852-f0153100-c43a-11eb-8233-16633cdec7c8.png)

**Comment by oonqt** [comment]
I need to switch to a different tracking API because of issues with the old one. I shut off the backend proxy for the time being I am in the process of finding one that returns all of the data that I need. I have been busy with finals & testing these past couple of weeks so I have not had much time to find & implement a different API provider. I should be able to investigate this next week and implement a fix on the backend, shouldn't require any app update.

**Comment by Willow-Systems** [comment]
Thanks for responding so quickly! No worries, just wanted to make sure you're aware.

Good luck with exams!

**Comment by Threnklyn** [comment]
Any updates on the issue?

**Comment by pauleeeeee** [comment]
Is the back-end still down?

**Comment by LOW-K3YS** [comment]
Any hope of this coming back online?

**Comment by oonqt** [comment]
> Any hope of this coming back online?

Yes. I'm getting back into Pebble myself, I just have to change my batteries and when I have the time I'll look into bringing this back, assuming someone else doesn't beat me to it... It's been a while since I've been in the programming space.
